benchmark.py

```diff
@@ -110,9 +108,7 @@ def benchmark(n):
         results, ["Table formatter", "time, μs", "rel. time"], "rst", floatfmt=".1f"
     )
 
-    import platform
```

Comment:
Also, revert this particular change as it is inconsistent with the general style followed elsewhere in this project. Revert the change and replace it with 'from platform import platform'